src/AppBundle/Resources/config/services.yml

```diff
@@ -57,3 +56,7 @@ services:
     AppBundle\Service\UserAgentMatcher:
       arguments:
         - '%mobile_app_user_agent_pattern%'
+
+
+    AppBundle\Controller\AuthenticationController:
+      $useGcmFallback: '%use_gcm_fallback%'
```

Comment:
I'd like to see the `userGcmFallback` argument wrapped in `arguments`, it might not be required, but for consistency sake..

Reply:
I've added the `bind:` option to be more precise and still using the full potential of autowiring.